.github/workflows/release.yml

```diff
@@ -210,6 +207,9 @@ jobs:
           cp bindings-x86_64-unknown-linux-gnu/* ./npm/linux-x64-gnu/
           cp bindings-x86_64-unknown-linux-musl/* ./npm/linux-x64-musl/
 
+      - name: Build Tailwind CSS
+        run: pnpm run build
+
```

Comment:
@thecrypticace Do you know if there was an inherent reason to doing the build step before we downloaded the artifacts? I move it further down so that when we build the standalone artifact we have access to everything.